src/inputs/IconCardGroup.tsx

```diff
@@ -9,29 +7,34 @@ import { useTestIds } from "src/utils";
 import { IconCard } from "src/inputs/IconCard";
 import { HelperText } from "src/components/HelperText";
 import { ErrorMessage } from "./ErrorMessage";
+import { Value } from "src/inputs";
+import { mergeProps, useField } from "react-aria";
+import { filterDOMProps } from "@react-aria/utils";
 
-export interface IconCardGroupItemOption {
+export interface IconCardGroupItemOption<V extends Value> {
   icon: IconProps["icon"];
   label: string;
   disabled?: boolean;
   /** The value of the IconCardGroup item, stored in value array in state. */
-  value: string;
+  value: V;
+  /** Exclusive: if true, this option will override all other options when selected. */
+  exclusive?: boolean;
```

Comment:
question: Do we need this 'exclusive' param?

Reply:
This is meant for options that need to be exclusive like "N/A"